src/python/director/transformUtils.py

```diff
@@ -205,3 +208,41 @@ def copyFrame(transform):
     t.PostMultiply()
     t.SetMatrix(transform.GetMatrix())
     return t
+
+
+# accepts vtkTransforms(). Outputs 6x6 matrix FM which represents the force-moment transformation
+# that is if wrench is 6 x 1 (moment, force) expressed in input frame, then corresponding wrench in
+# output frame would be numpy.dot(FM, wrench) = wrench_in_output_frame.
+def forceMomentTransformation(inputFrame, outputFrame):
+    FM = np.zeros((6,6))
+
+    inputToOutputFrame = copyFrame(inputFrame)
+    inputToOutputFrame.PostMultiply()
+    inputToOutputFrame.Concatenate(outputFrame.GetLinearInverse())
+
+    position, quaternion = poseFromTransform(inputToOutputFrame)
+    inputToOutputRotationMatrix = np.zeros((3,3))
+    vtk.vtkMath.QuaternionToMatrix3x3(quaternion, inputToOutputRotationMatrix)
+
+    FM[0:3, 0:3] = inputToOutputRotationMatrix
+    FM[3:,3:] = inputToOutputRotationMatrix
+
+    cross = crossProductMatrix(position)
+    
+    FM[0:3,3:] = np.dot(cross,inputToOutputRotationMatrix)
+
+    return FM
+
+# computes matrix P such that P*y = cross product of x and y
+def crossProductMatrix(x):
```

Comment:
I looked around the numpy documentation but couldn't find a method that provides this functionality.